lua/neotest-busted/_build_spec.lua

```diff
@@ -18,100 +16,67 @@ local function collect_node_names(node, names)
 	return collect_node_names(parent, names)
 end
 
+---@param tree neotest.Tree
+---@return string[]
+local function get_roots(tree)
+	local data = tree:data()
 
----Returns the name and configuration of the task the test file belongs to, if
----any.
----@param config    table   Busted configuration
----@param file_path string  Path to the test file
----@return string?, table?
-local function detect_task(config, file_path)
-	for k, v in pairs(config) do
-		local roots = v.ROOT
-		if roots and k ~= '_all' and k ~= 'default' then
-			for _, root in ipairs(roots) do
-				if lib.is_in_path(root, file_path) then
-					return k, v
-				end
-			end
-		end
+	if data.type ~= 'dir' then
+		return { data.path }
 	end
-end
 
----Given a directory path return a table which maps a task name to a list of
----all -roots of that task below the path
----@param config table
----@param path   string
----@return table<string, string[]>
-local function collect_tasks(config, path)
-	local function below_path(root)
-		return lib.is_in_path(path, root)
+	local paths = {}
+	for _, child in ipairs(tree:children()) do
+		local child_paths = get_roots(child)
+		vim.list_extend(paths, child_paths)
 	end
 
-	local result = {}
-	for k, v in pairs(config) do
-		local roots = vim.tbl_filter(below_path, v.ROOT or {})
-		if #roots > 0 then
-			result[k] = roots
-		end
-	end
-	return result
+	return paths
 end
 
 
 ---@param args neotest.RunArgs
 ---@return nil | neotest.RunSpec | neotest.RunSpec[]
-return function(args)
+local function build_spec(args)
 	local tree = args.tree
 	if not tree then return nil end
 	local data = tree:data()
 	local type = data.type
 
-	local command = vim.tbl_flatten {
+	local command = vim.iter {
 		vim.g.bustedprg or 'busted',
+	}:flatten():totable()
+
+	local additional_args = {
 		'--output',
-		require('neotest-busted._output-handler').source
+		require 'neotest-busted._output-handler'.source
```

Comment:
Please put the parentheses back. My style is that if `require` is part of a larger expression I write the parenthesis to make it look like any other inline function call.

```suggestion
		require('neotest-busted._output-handler').source
```